src/Resources/skeleton/command/Command.php.txt

```diff
@@ -22,6 +22,26 @@ class {{ command_class_name }} extends Command
         ;
     }
 
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        //Optional.
+        //If no initialization is required either leave this empty or remove it.
+
+        //This method is executed before the interact() and the execute() methods.
+        //Its main purpose is to initialize variables used in the rest of the command methods.
+    }
+
+    protected function interact(InputInterface $input, OutputInterface $output)
+    {
+        //Optional.
+        //If no interaction from the user is expected either leave this empty or remove it.
+
+        //This method is executed after initialize() and before execute().
+        //Its purpose is to check if some of the options/arguments are missing and interactively ask the user for those values.
+        //This is the last place where you can ask for missing options/arguments.
+        //After this command, missing options/arguments will result in an error.
```

Comment:
Can you move this to comments (`/**`) above the method?

And I think we can say less:

> This method is executed before execute() and can be used
> to interactively ask for information from the user
> https://symfony.com/doc/current/console.html#command-lifecycle

Reply:
I moved both comments outside of the methods and adjusted the contents like you suggested. 
Leaving both methods included for now. @javiereguiluz has a valid point concerning the overall simplicity of this bundle. Which could lead to complete removal of both additions anyways.